seleniumByTechnocredits/src/seleniumPractise/pallavi/Registration_Validation.java

```diff
@@ -0,0 +1,373 @@
+		aName.clear();
+		aName.sendKeys(address);
+		WebElement eName = driver.findElement(By.id("useremail"));
+		eName.sendKeys(email);
+		scroll();
+		ele.click();
+
+		Alert alt = driver.switchTo().alert();
+		Alert alt1 = driver.switchTo().alert();
+		Alert alt2 = driver.switchTo().alert();
+
+		if (alt.getText().equals("Please use correct email format")) 
+		{
+			System.out.println("Failure message is: " + alt.getText());
+			Thread.sleep(2000);
+			alt.accept();
+		} 
+		else if (alt1.getText().equals("Please use correct email format")) 
+		{
+			System.out.println("Failure message is: " + alt1.getText());
+			Thread.sleep(2000);
+			alt.accept();
+		} 
+		else 
+		{
+			alertMessage();
+		}
+		Thread.sleep(1000);
+		eName.clear();
+		Thread.sleep(1000);
+		eName.sendKeys("pallavi@gmail.com");
+		email = "pallavi@gmail.com";
+		ele.click();
+		alertMessage();
+	}
+
+	void cityN() throws InterruptedException 
+	{
+		WebElement fName = driver.findElement(By.id("fullName"));
+		fName.clear();
+		fName.sendKeys(fullName);
+		WebElement aName = driver.findElement(By.id("address"));
+		aName.clear();
+		aName.sendKeys(address);
+		WebElement eName = driver.findElement(By.id("useremail"));
+		eName.clear();
+		eName.sendKeys(email);
+		WebElement tName = driver.findElement(By.id("usercity"));
+		tName.sendKeys(city);
+		scroll();
+		ele.click();
+		alertMessage();
+	}
+
+	void companyN() throws InterruptedException 
+	{
+		WebElement fName = driver.findElement(By.id("fullName"));
+		fName.clear();
+		fName.sendKeys(fullName);
+		WebElement aName = driver.findElement(By.id("address"));
+		aName.clear();
+		aName.sendKeys(address);
+		WebElement eName = driver.findElement(By.id("useremail"));
+		eName.clear();
+		eName.sendKeys(email);
+		WebElement tName = driver.findElement(By.id("usercity"));
+		tName.clear();
+		tName.sendKeys(city);
+		WebElement cName = driver.findElement(By.id("organization"));
+		cName.sendKeys(companyName);
+		scroll();
+		ele.click();
+		alertMessage();
+	}
+
+	void nameOfUser() throws InterruptedException 
+	{
+		WebElement fName = driver.findElement(By.id("fullName"));
+		fName.clear();
+		fName.sendKeys(fullName);
+		WebElement aName = driver.findElement(By.id("address"));
+		aName.clear();
+		aName.sendKeys(address);
+		WebElement eName = driver.findElement(By.id("useremail"));
+		eName.clear();
+		eName.sendKeys(email);
+		WebElement tName = driver.findElement(By.id("usercity"));
+		tName.clear();
+		tName.sendKeys(city);
+		WebElement cName = driver.findElement(By.id("organization"));
+		cName.clear();
+		cName.sendKeys(companyName);
+		WebElement uName = driver.findElement(By.id("usernameReg"));
+		uName.sendKeys(userName);
+		ele.click();
+		if (userName.length() <= 5) 
+		{
+			Alert al = driver.switchTo().alert();
+			Thread.sleep(1000);
+			String alertMessage = al.getText();
+			Thread.sleep(1000);
+			al.accept();
+			String message = "Username length should be greater then 5 characters.";
```

Comment:
Hi Pallavi,

I believe after validating alert message then alert should get closed. So in If block of message validation accepting alert code should be written.